### PR TITLE
Mobile-menu 

### PR DESCRIPTION
**Pull request by coker2019**
Implemented mobile menu in my portfolio and used javascript to make it interactive.

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @coker2019 and Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

<div align='center'>

</div>


You have submitted a project not having the given linters configuration for a html/css/js project. Kindly ensure that the `.github/workflows/linters.yml` file is [exactly as given](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -1 +1,19 @@
...
+
+closed.addEventListener('click', () => {
+  mobileMenu.style.display = 'none';
+});
+
+mobileList.forEach((link) => {
+  link.addEventListener('click', () => {
+    mobileMenu.style.display = 'none';
+  });
+});
```
Comment (aamirkhan2478) [inline index.js]: - [ ] Well done :clap:  you did very nice work :+1: but your code is not `DRY` please make sure it is `DRY` which makes it cleaner and readable. `DRY` stand for `Don't Repeat Yourself`, a fundamental principle of software development aimed at reducing the repetition of information. Check below :point_down:  my suggestions :slightly_smiling_face: 

 ```suggestion
const hamburgerButton = document.querySelector('.hamburger');
const mobileMenu = document.querySelector('.mobile-lists');
const closed = document.querySelector('.close-icon');
const mobileList = document.querySelectorAll('.mobile-list');

function toggleMobileMenu() {
  if (mobileMenu.style.display === 'block') {
    mobileMenu.style.display = 'none';
  } else {
    mobileMenu.style.display = 'block';
  }
}

hamburgerButton.addEventListener('click', (event) => {
  toggleMobileMenu();
  event.preventDefault();
});

closed.addEventListener('click', toggleMobileMenu);

mobileList.forEach((link) => {
  link.addEventListener('click', toggleMobileMenu);
});
```
- [ ] In this refactored code, we create a `toggleMobileMenu` function that checks the current display state of the mobile menu and toggles it accordingly. We then use this function in the event listeners for the hamburger button, the close button, and the mobile list links.

- [ ] This approach reduces repetition and makes the code more maintainable, as any changes to the show/hide behaviour of the mobile menu can be made in one place.

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @coker2019 , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linter passed

## Required Changes ♻️
- [ ] Your `Portfolio` link is  not working kindly fix it when someone clicks on `portfolio` from your navbar it should scroll down to the `Portfolio` section I think you missed the `id`  attribute in the `Portfolio` section :+1: 

_**Hint**_
```html
<section class="works" id='works-sec'>
```

- [ ] Please stick to the Figma design your `cross` icon is not the same as the Figma design so kindly fix this Figma design. Check below :point_down: the comparison your design and FIgma design :slightly_smiling_face: 

**Your design**
Screenshot from 2023-04-17 17-57-09

**Figma Design**
Screenshot from 2023-04-17 17-57-17


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @coker2019 and @akezeth , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Use JavaScript events ✔️
Made required changes ✔️ 

## Optional suggestions

- [optional] Please kindly format your README file in a more professional way; you can use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md). Ensure that the missing headers are added (Tech stack and show your support) and Acknowledgement header should not have an empty content. Kindly remove the template text from the license header as well. 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**